components/Post.js

```diff
@@ -119,7 +120,7 @@ export const Post = (post) => {
 		if (!post.end) {
```

Comment:
i think the purpose of this initially was to draw dividers under each post. as far as I know we don't need any of this code anymore

Reply:
Which piece is not needed anymore?

Reply:
the entire `if(!post.end)` and its contents